src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -71,12 +71,19 @@ public AntiForgeryOptions AntiForgeryOptions
 
         /// <summary>
         /// Gets a list of <see cref="ExcludeValidationDescriptor"/> which are used to construct a list
-        /// of exclude filters by <see cref="IValidationExcludeFiltersProvider"/>,
+        /// of exclude filters by <see cref="IValidationExcludeFiltersProvider"/>.
         /// </summary>
         public List<ExcludeValidationDescriptor> ValidationExcludeFilters { get; }
             = new List<ExcludeValidationDescriptor>();
 
         /// <summary>
+        /// Gets a list of <see cref="CacheProfile"/> which are pre-defined settings for
+        /// <see cref="ResponseCacheFilter"/>.
+        /// </summary>
+        public List<CacheProfile> CacheProfiles { get; }
```

Comment:
If we remove `Name` property then `CacheProfile` would make sense ONLY as a Dictionary. Individually, each profile will be a set of values without a key. So I think we should just have the `Name` property as is and expose only `Add` and `Get` functions from the list.